clients/src/test/java/org/apache/kafka/common/requests/LeaderAndIsrRequestTest.java

```diff
@@ -59,18 +61,41 @@ public void testUnsupportedVersion() {
 
     @Test
     public void testGetErrorResponse() {
-        Uuid id = Uuid.randomUuid();
+        Uuid topicId = Uuid.randomUuid();
+        String topicName = "topic";
+        int partition = 0;
+
         for (short version = LEADER_AND_ISR.oldestVersion(); version < LEADER_AND_ISR.latestVersion(); version++) {
```

Comment:
It does not cover all versions. I had filed a PR for that (#10078) but I feel it can be fixed here.

Reply:
Good catch!